src/organisation/containers/update-pba-check/update-pba-numbers-check.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import { Organisation } from 'src/organisation/organisation.model';
+import { OrganisationDetails } from '../../../models/organisation.model';
+import * as fromStore from '../../store';
+
+@Component({
+  selector: 'app-prd-update-pba-numbers-check-component',
+  templateUrl: './update-pba-numbers-check.component.html',
+})
+export class UpdatePbaNumbersCheckComponent implements OnInit {
+
+  public readonly title: string = 'Check your PBA accounts';
```

Comment:
Query over `accounts` vs `numbers`?

Reply:
Resolving comment since it's been confirmed the UX team were happy with "PBA accounts".